nikola/nikola.py

```diff
@@ -2183,6 +2183,7 @@ def generic_page_renderer(self, lang, post, filters, context=None):
         context['title'] = post.title(lang)
         context['description'] = post.description(lang)
         context['permalink'] = post.permalink(lang)
+        context['crumbs'] = utils.get_crumbs(post.destination_path(lang, extension))
```

Comment:
This may override another `crumbs` variable in the context.  Perhaps do `if 'crumbs' not in context`?